runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -72,7 +72,7 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 	MM_StandardAccessBarrier(MM_EnvironmentBase *env, MM_MarkingScheme *markingScheme) :
 		MM_ObjectAccessBarrier(env)
 #if defined(J9VM_GC_GENERATIONAL)
-		, _scavenger(NULL)
+		, _scavenger(_extensions->scavenger)
 #endif /* J9VM_GC_GENERATIONAL */
 		, _markingScheme(markingScheme)
```

Comment:
we could probably get it from ext->globalCollector->markingScheme, to avoid passing through constructor to be consistent how we get scavenger, but this is still ok

Reply:
I can change it in the next cleaning PR